Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
We already say "in certain games", so "which leads to" is fine IMO. "bugs and freezes" is correct, because you can get freezes in some parts of a game and (non-freeze) bugs in other parts.